documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc

```diff
@@ -201,6 +201,26 @@ Both the local and the backup call(if the "backupFailurePolicy" is set to "FAIL"
 ===== Asynchronous local cluster
 In the case of asynchronous local clusters, the backup data is sent during the commit phase. If the backup call fails and the "backupFailurePolicy" is set to "FAIL" then the user is notified through an exception.
 
+==== Replication of expired cache entries across sites
+You can configure {brandname} to expire entries, which controls the amount of
+time entries remain in the cache.
+
+* `lifespan` expiration is suitable for cross-site replication because entries
+expire without the need for {brandname} to determine if the entries have
+expired in clusters on other sites.
+
+* `max-idle` expiration does not work with cross-site replication because
+{brandname} cannot determine if cache entries have reached the idle timeout in
+clusters on other sites.
+
+[IMPORTANT]
+====
+When expiring entries, {brandname} replicates commands across sites for both
+the `lifespan` and `max-idle` parameters. However, these commands have no
```

Comment:
This isn't quite correct. It is more along the lines that it replicates a `remove expired` command that removes the entry from all nodes in the cluster. I wouldn't worry about talking about lifespan or max-idle. Then you can note that this command is currently replicated to cross sites, which is unnecessary. Can we mention the replication will be removed in the future?

Reply:
@wburns thanks for that. hopefully the updates make it clear now.